Source/WebCore/page/LocalDOMWindow.idl

```diff
@@ -68,7 +68,7 @@
     readonly attribute Navigator navigator;
     [Replaceable, ImplementedAs=navigator] readonly attribute Navigator clientInformation;
     // FIXME: This is specified to be [SecureContext]
-    [EnabledByQuirk=shouldEnableApplicationCache] readonly attribute DOMApplicationCache applicationCache;
+    readonly attribute DOMApplicationCache applicationCache;
```

Comment:
This change enables it everywhere.  Let's just change Quirks::shouldEnableApplicationCacheQuirk to return false unconditionally in this PR.